### DISCUSSION TITLE
Is there a way to increase the line thickness?

### DISCUSSION THREAD

**Discussion by rosmur** [markmap/markmap#162] [Q&A]
I'd like to increase/customize the line thickness. Is it possible to do this? 

Also, would like to have the ability to edit colors individually and increase spacing etc. Is that possible?

**Comment by gera2ld** [comment]
It's possible but you need to write code by yourself.
It's not possible if you are using the web demo or VSCode extension.

**Reply by rosmur** [reply]
Any pointers on what part of the code to look at?

**Reply by gera2ld** [reply]
https://github.com/markmap/markmap/blob/acc6482efd58274bb2703dbae2ed218ea8d8f3a9/packages/markmap-view/src/view.tsx#L90
